module-gui/gui/widgets/text/core/lines/ScopedParentDisown.hpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <Item.hpp>
+
+namespace gui
+{
+    /// class to disown Item temporary to ignore callback
```

Comment:
don't we already have such class somewhere in UI? :) I assume somewhere in Layouts code? :)

Reply:
Will check but I do not recall -> I have just moved it from TextLine to separate cpp.

Reply:
Can't find any other -> i believe that was constructed for Text.